packages/jellyfish-api-core/src/category/vault.ts

```diff
@@ -344,6 +360,13 @@ export interface ListAuctionHistoryDetail {
   auctionWon: string[]
 }
 
+export interface VaultEstimation {
+  collateralValue: string // n.nnnnnnnn (amount) The total collateral value in USD
```

Comment:
I agree that `string` > `BigNumber` > `number` as string handle precision better than the other two
If you see whale endpoint, BigNumber is written as String.
However, in this context, the type should be the RETURN TYPE, which is BigNumber.
You may take a look at other RPC files